test/__snapshots__/cpa.test.js.snap

```diff
@@ -38,3 +38,37 @@ exports[`should print no matches found 1`] = `
 "No matches found for nomatch.js
 "
 `;
+
```

Comment:
Its caused because of the path mocking in jest. Fixing and cleaning up #3 will make this easier and avoid the path mocking.